experiments/AMIP/modular/components/ocean/eisenman_seaice.jl

```diff
@@ -0,0 +1,293 @@
+import ClimaCoupler: FluxCalculator
+import ClimaTimeSteppers as CTS
+import DiffEqBase: ODEProblem, init, step!
+import ClimaCoupler.FluxCalculator: update_turbulent_fluxes_point!, differentiate_turbulent_fluxes!
+import ClimaCoupler.Interfacer: get_field, update_field!
+
+# TODO: move to ClimaCore
+import Base.getindex
+Base.getindex(f::Fields.FieldVector, colidx) =
+    Fields.FieldVector(; zip(propertynames(f), map(x -> getproperty(f, x)[colidx], propertynames(f)))...)
+
+"""
+    EisenmanIceSimulation{P, Y, D, I}
+
+Thermodynamic 0-layer, based on the Semtner 1979 model and later refined by  sea ice model as specified by
+Eisenmen 2009 and Zhang et al 2021.
+"""
+struct EisenmanIceSimulation{P, Y, D, I} <: SeaIceModelSimulation
+    params_ice::P
+    Y_init::Y
+    domain::D
+    integrator::I
+end
+name(::EisenmanIceSimulation) = "EisenmanIceSimulation"
+
+Base.@kwdef struct EisenmanIceParameters{FT <: AbstractFloat}
+    h::FT = 2                       # sea ice height [m]
```

Comment:
I may just be ignorant about what a "parameter" means, but the ice thickness is a prognostic variable right? Do we re-build this struct each time-step? (Maybe I will find out later on...)

Reply:
No you're right, ice thickness, `h_ice`, is a prognostic variable. The first three params here were leftover from testing, but they are not actually being used. This is now removed. Thanks for spotting this!